testutils/emptypkg: TestDockerCLI failed

[(testutils/emptypkg).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2058043&tab=buildLog) on [master@17c649468273e3c80953681e90fc997f0d75d2dd](https://github.com/cockroachdb/cockroach/commits/17c649468273e3c80953681e90fc997f0d75d2dd):

```
=== RUN   TestDockerCLI
    TestDockerCLI: test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI312336098
    TestDockerCLI: test_log_scope.go:58: use -show-logs to present logs inline
--- FAIL: TestDockerCLI (378.45s)
=== RUN   TestDockerCLI/test_high_verbosity.tcl
    --- FAIL: TestDockerCLI/test_high_verbosity.tcl (33.15s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/testutils/emptypkg TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(testutils/emptypkg).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2060379&tab=buildLog) on [master@a80acfa81ca50e721b4f1feb9ecdd1f404aa48f1](https://github.com/cockroachdb/cockroach/commits/a80acfa81ca50e721b4f1feb9ecdd1f404aa48f1):

```
=== RUN   TestDockerCLI
    TestDockerCLI: test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI036488094
    TestDockerCLI: test_log_scope.go:58: use -show-logs to present logs inline
--- FAIL: TestDockerCLI (349.55s)
=== RUN   TestDockerCLI/test_demo_partitioning.tcl
    --- FAIL: TestDockerCLI/test_demo_partitioning.tcl (43.17s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/testutils/emptypkg TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(testutils/emptypkg).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2060516&tab=buildLog) on [master@1b6f9d0e566372a24b54593d6c998d09c9adb518](https://github.com/cockroachdb/cockroach/commits/1b6f9d0e566372a24b54593d6c998d09c9adb518):

```
=== RUN   TestDockerCLI
    TestDockerCLI: test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI346264608
    TestDockerCLI: test_log_scope.go:58: use -show-logs to present logs inline
--- FAIL: TestDockerCLI (378.19s)
=== RUN   TestDockerCLI/test_demo_partitioning.tcl
    --- FAIL: TestDockerCLI/test_demo_partitioning.tcl (61.99s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/testutils/emptypkg TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(testutils/emptypkg).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2060948&tab=buildLog) on [master@9224ec00bc0e5d953cddaa746adf2bcb3bc253b7](https://github.com/cockroachdb/cockroach/commits/9224ec00bc0e5d953cddaa746adf2bcb3bc253b7):

```
=== RUN   TestDockerCLI
    TestDockerCLI: test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI140216181
    TestDockerCLI: test_log_scope.go:58: use -show-logs to present logs inline
--- FAIL: TestDockerCLI (350.87s)
=== RUN   TestDockerCLI/test_demo_partitioning.tcl
    --- FAIL: TestDockerCLI/test_demo_partitioning.tcl (44.25s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/testutils/emptypkg TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/kv/kvclient/kvcoord/range_iter.go
+++ b/pkg/kv/kvclient/kvcoord/range_iter.go
@@ -174,10 +174,6 @@ func (ri *RangeIterator) Seek(ctx context.Context, key roachpb.RKey, scanDir Sca
 	for r := retry.StartWithCtx(ctx, ri.ds.rpcRetryOptions); r.Next(); {
 		rngInfo, err := ri.ds.getRoutingInfo(ctx, ri.key, ri.token, ri.scanDir == Descending)
 
-		if log.V(2) {
-			log.Infof(ctx, "key: %s, desc: %s err: %v", ri.key, rngInfo.Desc(), err)
-		}
-
 		// getRoutingInfo may fail retryably if, for example, the first
 		// range isn't available via Gossip. Assume that all errors at
 		// this level are retryable. Non-retryable errors would be for
@@ -187,11 +183,11 @@ func (ri *RangeIterator) Seek(ctx context.Context, key roachpb.RKey, scanDir Sca
 			log.VEventf(ctx, 1, "range descriptor lookup failed: %s", err)
 			continue
 		}
+		if log.V(2) {
+			log.Infof(ctx, "key: %s, desc: %s", ri.key, rngInfo.Desc())
+		}
 
 		ri.token = rngInfo
-		if log.V(2) {
-			log.Infof(ctx, "returning; key: %s, desc: %s", ri.key, ri.Desc())
-		}
 		return
 	}
 

